mbs-s0014-20230920/README.md

```diff
@@ -0,0 +1,98 @@
+# Map-based noise simulations based on the SAT `mbs-noise-sims-sat` time-domain simulations
+
+Tag: `mbs-s0014-20230920`
+
+## Updates
+
+* 2023-09-20: first release.
+
+## Summary
+
+300 realistic realizations of the SAT map noise, estimated from the [mbs-noise-sims-sat](https://github.com/simonsobs/pwg-scripts/tree/master/pwg-tds/mbs-noise-sims-sat) time-domain noise simulations. Simulations cover the full SAT footprint and utilize the tiled noise model.
+
+## Noise model
+
+This release is based on the [mbs-noise-sims-sat](https://github.com/simonsobs/pwg-scripts/tree/master/pwg-tds/mbs-noise-sims-sat) time-domain simulations, available on `NERSC` at:
+
+    /global/cfs/cdirs/sobs/sims/sat-noise-sims/v1
+
+These time-domain simulations include detector and atmospheric noise scaled to match a full year of sensitivity split into two disjoint maps for each frequency band. 
+
+Using [`mnms`](https://github.com/simonsobs/mnms), we generate 300 realizations of the noise using map-based methods. Specifically, we use the tiled model, governed by the configured [parameters](parameters/so_sat_v1_f1.yaml).
+
+Notes:
+* Both time-domain and map-domain simulations are stored in HEALPix format. See [examples](#example-usage) below for further guidance.
+* A mask has been applied to the map-domain simulations. The masks are stored here: `/global/cfs/cdirs/sobs/v4_sims/mbs/mbs_s0014_20230920/car/masks/coadd_SAT_{bands}_mask_obs_hp.fits`.
+* See these [slides](https://drive.google.com/file/d/1ZkwpjYQVCkmbRk5RWOp0Qq4CPsynBH31/view?usp=sharing) for validation of the map-based simulations.
+
+## Available maps
+
+Maps are available in the HEALPix format. Maps are in Equatorial Coordinates, `K_CMB` units, FITS format.
+
+The maps are available on `NERSC` at:
+
+    /global/cfs/cdirs/sobs/v4_sims/mbs/mbs_s0014_20230920/out/run00
```

Comment:
it would be interesting to add how much space they occupy

Reply:
Added! It's 215 GB